crates/nixci/src/lib.rs

```diff
@@ -55,6 +60,29 @@ pub async fn nixci(
     }
 }
 
+async fn remote_build(
+    cmd: &NixCmd,
+    build_cfg: &BuildConfig,
+    cfg: &config::Config,
+    host: &String,
+) -> anyhow::Result<Vec<StorePath>> {
+    let omnix_input = format!("{}", env!("OMNIX"));
+
+    let metadata = nix_rs::flake::metadata::get_flake_metadata_json(cmd, &cfg.flake_url.0).await?;
+
+    nix_rs::copy::run_nix_copy(
+        cmd,
+        &host,
+        &omnix_input,
+        &metadata.path,
+        &build_cfg.extra_nix_build_args,
+    )
+    .await?;
```

Comment:
You need to `nix copy` the paths in `--override-inputs` too (if specified).

Reply:
Similar to:

https://github.com/srid/nixos-flake/blob/5734c1d9a5fe0bc8e8beaf389ad6227392ca0108/activate/activate.nu#L103-L107

Reply:
I handled it in this way.
(https://github.com/juspay/omnix/pull/212/files#diff-356b04180edb27abcd766667939d7a544466fdc67df4005f041453532a9a87e8R16-R29)

Reply:
It shouldn't be done that way. See this link for why:

https://en.wikipedia.org/wiki/Separation_of_concerns

Reply:
Also, the logic is wrong.

Reply:
Because, we are supposed to run `nix copy` **for each** entry in `--overrride-inputs` list.

Reply:
> Also, the logic is wrong.

I could be wrong, though (is `nix copy` automagically copying those paths?). Need to test to know for sure.

Reply:
Interesting, looks like it *does* automagically copy it? 

<img width="969" alt="image" src="https://github.com/user-attachments/assets/306c9ba4-1a1a-4605-a040-af65c9fb0059">

@rsrohitsingh682 Can you make it so that the stdout of `nix copy` is shown in omnix console output as well?